src/FoundryCSS/ui/blocks/Button/_button.scss

```diff
@@ -10,16 +10,20 @@
 %button,
 .button {
     display: inline-block;
-    padding: map-get($button-sizes, 'normal');
+    padding: map-get($button-sizes, 'default');
     border-radius: $button-border-radius;
     text-decoration: none;
     letter-spacing: $button-label-leading;
     font-weight: $button-label-weight;
     cursor: pointer;
     text-align: center;
-    border: 2px solid transparent;
-    background-color: color(base);
-    color: color(white);
+    border: 2px solid color('default');
+    background-color: color('default');
+    color: color('white');
+
+    &.button--bordered {
+        color: color('default');
+    }
 
     %_is-busy {
```

Comment:
How would you extend this? Or is it automatic if you extend the top level class with the percentage syntax?

Reply:
You can extend the top level class but the child selectors must be .classes otherwise they are ignored. Makes sense - %placeholders don't exist unless extended.

Reply:
So does `%_is-busy` ever get extended, as I can only see a placeholder selector

Reply:
Not on it's own no it's intended to be a state of .button/%button. I have just changed this file so these are all .classes though otherwise they're essentially useless?